tests/unit/api.json

```diff
@@ -47,6 +47,18 @@
                 "VAL2",
                 "VAL3"
               ]
+            },
+            {
+              "name": "stream_enum",
+              "description": "Whether to return the response as a stream_object",
+              "required": true,
+              "allowMultiple": false,
+              "type": "string",
```

Comment:
i was about to suggest to use "boolean". but ironically, it turns out that the "string" type with enum contraints is indeed simpler.

Reply:
Happy to use boolean if you prefer. I actually used enum since I did a quick check and dind't see any existing uses of boolean but looking at seastar-json2code.py a `boolean` type is indeed supported.

Reply:
I think it's less surprising to have a boolean here.

Reply:
I changed it to a boolean in c9f848b1d diff: https://github.com/scylladb/seastar/compare/9f222060c1e51bc2c96cc8b2a49023b140d00eae..c9f848b1d81f616162787b67b2a55b921f20c0ba